server/api/graphql_run.go

```
@@ -135,7 +135,12 @@ func (r *RunResolver) Metadata(ctx context.Context) (*MetadataResolver, error) {
 }
 
 func (r *RunResolver) Playbook(ctx context.Context) (*PlaybookResolver, error) {
-	return getGraphqlPlaybook(ctx, r.PlaybookID)
+	val, err := getGraphqlPlaybook(ctx, r.PlaybookID)
+	if err != nil {
+		return nil, nil
+	}
```

It feels weird to just swallow this error. Might we check explicitly for a permissions error and ignore that explicitly?

---

Yes it does feel strange. We can explicit check permissions errors for this reason. Decided to log otherwise.